# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu_bindings/moz.yaml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 gfx/wgpu_bindings/moz.yaml
--- a/gfx/wgpu_bindings/moz.yaml	Fri Oct 03 18:01:30 2025 +0000
+++ b/gfx/wgpu_bindings/moz.yaml	Fri Oct 03 18:08:29 2025 +0000
@@ -8,8 +8,8 @@
   name: wgpu
   description: A cross-platform pure-Rust graphics API, modeled on the WebGPU standard
   url: https://github.com/gfx-rs/wgpu
-  release: commit ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24
-  revision: ffd4e9fb53a98f8ad67bab8c5d2b42cc9dbdff24
+  release: 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec (2025-09-29T17:15:32Z).
+  revision: 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec
   license: ['MIT', 'Apache-2.0']
 
 updatebot: